test/e2e/common_suite_test.go

```diff
@@ -345,15 +345,19 @@ func (tc *testCase) run() {
 				}
 				log.Infof("Deleting pod %s...", tc.pod.Name)
 
-				if tc.deletionWithin != nil {
-					if err = wait.For(conditions.New(
-						client.Resources()).ResourceDeleted(tc.pod),
-						wait.WithInterval(5*time.Second),
-						wait.WithTimeout(*tc.deletionWithin)); err != nil {
-						t.Fatal(err)
-					}
-					log.Infof("Pod %s has been successfully deleted within %.0fs", tc.pod.Name, tc.deletionWithin.Seconds())
+				// Add a default deleteWithin time.
+				if tc.deletionWithin == nil {
+					deleteWithin := time.Duration(time.Second * 300)
```

Comment:
```suggestion
					deleteWithin := time.Second * 300
```
I think `time.Second` is already of type `time.Duration`?

Reply:
@surajssd any thoughts about this? otherwise lgtm.

Reply:
Ah my bad, I missed this one.

Reply:
Done now.